zerver/management/commands/rundjangoserver.py

```diff
@@ -1,13 +1,13 @@
-from datetime import datetime
 from functools import wraps
 from typing import Callable
 
 from django.core.management.commands.runserver import Command as DjangoCommand
+from django.utils.timezone import now as timezone_now
 
 
 def output_styler(style_func: Callable[[str], str]) -> Callable[[str], str]:
     # Might fail to suppress the date line around midnight, but, whatever.
-    date_prefix = datetime.now().strftime("%B %d, %Y - ")
+    date_prefix = timezone_now().strftime("%B %d, %Y - ")
```

Comment:
I think the reason we had this in `zilencer` was the same reason we have `populate_db` in that directory -- because it's unsafe to run this management command in production, and we'd like to make that hard to do.  (Though of course our current mechanism only blocks this for folks using release tarballs, that's most of the userbase).

I think the right fix would be to create a separate `development` app, and move the half of `zilencer/management/commands/` that is there for that reason to this new app. 

`zproject/dev_urls.py` and `zerver/views/development/` would also move to the new app.

Reply:
I'm going to drop this commit from the series, and we can circle back to it later.  I like the concept of disentangling our dev env from the "I'm a push bouncer" app.